chain/network/src/peer.rs

```diff
@@ -171,10 +171,8 @@ impl Peer {
     }
 
     fn fetch_client_chain_info(&mut self, ctx: &mut Context<Peer>) {
-        ctx.wait(self.client_addr
-            .send(NetworkClientMessages::GetChainInfo)
-            .into_actor(self)
-            .then(move |res, act, _ctx| match res {
+        ctx.wait(self.client_addr.send(NetworkClientMessages::GetChainInfo).into_actor(self).then(
```

Comment:
hm, reformat back?

Reply:
This against the formatter will. I think we all should use same version of rustfmt.